keyboards/keyquest/enclave/keymaps/default/keymap.c

```diff
@@ -0,0 +1,70 @@
+#include QMK_KEYBOARD_H
+
+/* enum custom macros */
+enum custom_keycodes {
+    COPY = SAFE_RANGE,
+    PASTE,
+    WINCLIP,
+};
+
+/* custom macros */
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+
+    case COPY:
+        if (record->event.pressed) {
+           SEND_STRING(SS_LCTL("c"));
+        }
+        break;
+
+    case PASTE:
+        if (record->event.pressed) {
+           SEND_STRING(SS_LCTL("v"));
+        }
+        break;
+
+    case WINCLIP:
+        if (record->event.pressed) {
+           SEND_STRING(SS_LGUI("v"));
+        }
+        break;
+    }
+    return true;
+};
```

Comment:
No need for any of this.  

`C(KC_C)`, `C(KC_V)` and `G(KC_V)` can be used in the keymap directly, without the need for custom macros.

Reply:
This was done intentionally to have a template to move onto more complex macros. If it needs to be changed i will update but this is the ideal default keymap